core/src/test/java/io/nixer/nixerplugin/core/detection/registry/FailedLoginRatioRegistryTest.java

```diff
@@ -22,31 +23,41 @@
 
     private FailedLoginRatioRegistry registry;
 
+    private final Instant baseInstant = Instant.ofEpochSecond(600);
```

Comment:
It would be clearer to make it a constant called `NOW` as the current name is not much helpful.